llvm/include/llvm/SYCLLowerIR/LowerESIMD.h

```diff
@@ -34,6 +34,9 @@ class SYCLLowerESIMDPass : public PassInfoMixin<SYCLLowerESIMDPass> {
 FunctionPass *createSYCLLowerESIMDPass();
 void initializeSYCLLowerESIMDLegacyPassPass(PassRegistry &);
 
+FunctionPass *createESIMDLowerLoadStorePass();
```

Comment:
Pass is written to work with old pass manager only. Please support it for the new pass manager, SYCLLowerESIMDPass or other pass can be used as an example.